docs/getting_started.md

```diff
@@ -24,7 +24,7 @@ Download the OpenShift pull secret from the https://console.redhat.com/openshift
 
 ## Bootstrap MicroShift
 
-Run the following commands to initiate the creation process of the `microshift-starter` virtual machine with 2 CPU cores, 2GB RAM and 20GB storage.
+Run the following commands to initiate the creation process of the `microshift-starter` virtual machine with 2 CPU cores, 3GB RAM and 20GB storage.
 
 ```bash
 VMNAME=microshift-starter
```

Comment:
> this might be a bit confusing to someone following the instructions.

I noticed that the hostnames do get fixed later on, so maybe just a note explaining not to worry about it would be sufficient.

Reply:
I fixed this "problem" in the current PR. 
Please, see [this diff](https://github.com/openshift/microshift/pull/1844/files#diff-c4dab62439b17e2dbab38be07781beba8553f8d4a5654b4247fa54956be09d93) - the host name is no longer set in the kickstart.